src/RequestManager.ts

```diff
@@ -49,12 +52,23 @@ class RequestManager {
   }
   private onData(data: string): void {
     const parsedData = JSON.parse(data);
-    if (typeof parsedData.result === "undefined") {
+    if (typeof parsedData.result === "undefined" && typeof parsedData.error === "undefined") {
```

Comment:
is this right do all rpc request require an error field to be defined?

Reply:
as an aside...
We should get some of these standard interfaces together
```
type TJSONRPCPayload = { ... }
...
```

Reply:
discussed offline, its checking for the noop case.